ember-cli-build.js

```diff
@@ -2,48 +2,11 @@
 
 const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 
-const environment = process.env.EMBER_ENV;
-const IS_PROD = environment === 'production';
-const IS_TEST = environment === 'test';
-
 module.exports = function (defaults) {
   let app = new EmberApp(defaults, {
-    autoImport: {
-      exclude: ['flandersMunicipalities.ts'],
-      webpack: {
-        node: {
-          global: true,
```

Comment:
The build didn't seem to fail with this removed, so I don't think we need it anymore.